### PR TITLE
Add capability to choose to store files or not

### PR DESCRIPTION
**Pull request by sebprev**
Add capability to choose if it should store files or not. 
To do this, file paths descriptions in config file is optionnal. If set, we have to store files in paths, if not, we don't store anything.

### CODE REVIEW

File: src/main/java/org/openas2/processor/storage/MDNFileModule.java [hunk 1]
```diff
@@ -29,15 +29,18 @@ public void handle(String action, Message msg, Map options) throws OpenAS2Except
 			throw new OpenAS2Exception("Message has no MDN");
 		}
 
-		try
+		if (getParameter(PARAM_FILENAME, false) != null)
```
Comment (fperie) [inline src/main/java/org/openas2/processor/storage/MDNFileModule.java]: You can write:

String paramFileName = getParameter(PARAM_FILENAME, false);
if (paramFileName != null)
{
      File mdnFile = getFile(msg, paramFileName, "");
      try(InputStream in = getMDNStream(msg.getMDN()))
      {
        store(mdnFile, in);
      }
}

File: src/main/java/org/openas2/processor/storage/MessageFileModule.java [hunk 2]
```diff
@@ -32,18 +32,21 @@
...
-			File msgFile = getFile(msg, getParameter(PARAM_FILENAME, true), action);
-			InputStream in = msg.getData().getInputStream();
-			store(msgFile, in);
-			LOGGER.info("stored message to {}", msgFile.getAbsolutePath() + msg.getLoggingText());
-		}
-		catch (Exception e)
-		{
-			throw new DispositionException(new DispositionType("automatic-action", "MDN-sent-automatically",
-					"processed", "Error", "Error storing transaction"), AS2ReceiverModule.DISP_STORAGE_FAILED, e);
+			// store message content
```
Comment (fperie) [inline src/main/java/org/openas2/processor/storage/MessageFileModule.java:38]: the same previous comment

**Comment by fperie** [comment]
Please, can you describe your modification in the CHANGELOG and README files ?
Else, very good work !

**Review by fperie — COMMENTED** [review]
Please, can you describe your modification in the CHANGELOG and README files ?
Else, very good work !

**Comment by sebprev** [comment]
I agree comments. 

In the README file i'll change the GAV with 1.3.3-SNAPSHOT even though SNAPSHOT version won't be available on Maven Central.
While you won't release 1.3.3 the README file won't be correct.

Or maybe you prefer i don't modify it for now ? Or directly in 1.3.3 released version ?

**Comment by fperie** [comment]
The maven release plugin don't touch the README.md file.
Please, can you putting the 1.3.3 released version

**Comment by sebprev** [comment]
Done.

**Review by fperie — APPROVED** [review]

**Pull request merged**